sql/core/src/test/resources/sql-tests/results/string-functions.sql.out

```diff
@@ -1136,3 +1136,12 @@ struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
 Invalid value for the 'format' parameter of function 'to_binary': invalidformat. The value has to be a case-insensitive string literal of 'hex', 'utf-8', or 'base64'.
+
+
+-- !query
+select to_binary('a!', 'base64')
+-- !query schema
+struct<>
+-- !query output
+java.lang.IllegalArgumentException
```

Comment:
We should migrate this on Spark's exception. Do we have a ticket for that?

Reply:
This is the behavior of the function `Encode`, which is introduced since Spark 1.5